R/paste_linter.R

```diff
@@ -21,6 +21,40 @@
 #'   `paste()` with `sep = ""` is not linted.
 #' @param allow_to_string Logical, default `FALSE`. If `TRUE`, usage of
 #'   `paste()` and `paste0()` with `collapse = ", "` is not linted.
+#'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "paste('a', 'b', sep = '')",
+#'   linters = paste_linter()
+#' )
+#'
+#' lint(
```

Comment:
please add examples for cases 3 & 4, e.g. `paste0("a", "b", sep = " ")` and `paste(rep("x", 20L), collapse = "")`